fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/service/LoanWritePlatformServiceJpaRepositoryImpl.java

```diff
@@ -1350,8 +1349,6 @@ public CommandProcessingResult chargebackLoanTransaction(final Long loanId, fina
 
         handleChargebackTransaction(loan, newTransaction);
 
-        newTransaction = this.loanTransactionRepository.saveAndFlush(newTransaction);
```

Comment:
if new transaction is not saved and flushed, there will be no id in the `businessEventNotifierService.notifyPostBusinessEvent(new LoanChargebackTransactionBusinessEvent(newTransaction));`